angular-spa-ui/js/search-app/search-module/index.js

```diff
@@ -1,15 +1,22 @@
 "use strict";
 
-require('angular-ui-router');
+module.exports = function(ngModule) {
+    
+    var advancedSearchModule = require('./advanced-search')(ngModule);
+    var searchModule = ngModule.module('searchApp.searchModule', ['ui.router', 'searchApp.searchModule.advancedSearch']);
```

Comment:
2 Check do you really need "searchApp.searchModule.advancedSearch" for normal running the aplpication